src/languages/en.js

```diff
@@ -267,6 +267,7 @@ export default {
         deleteWorkspaceError: 'Sorry, there was an unexpected problem deleting your workspace avatar.',
         sizeExceeded: ({maxUploadSizeInMB}) => `The selected image exceeds the maximum upload size of ${maxUploadSizeInMB}MB.`,
         tooSmallResolution: ({minHeightInPx, minWidthInPx}) => `Please upload an image larger than ${minHeightInPx}x${minWidthInPx} pixels`,
+        notAllowedExtension: 'Profile picture must be one of the following types:',
```

Comment:
```suggestion
        notAllowedExtension: ({allowedExtensions}) => 'Profile picture must be one of the following types: ${allowedExtensions.join(", ")}',
```

This error is incomplete with out the types, so lets push the types in the message.